doc/quickstart.rst

```diff
@@ -0,0 +1,294 @@
+Quickstart
+==========
+
+Let's learn Ralph by example. Throughout this tutorial, we’ll walk you through:
+
+- adding asset information, licenses and support
+- scanning asset details and configuration
+- access billing information
+
+
+Assets
+^^^^^^
+
+Searching and Filtering of Assets
+---------------------------------
+
+Let's start with the main screen. Here you find all your hardware assets which
+are in the database. Use the left column for filtering of the assets.
+
+There are two types of assets -- devices and parts. A device could be a blade
+server, and a part is a component of this server, for example memory or hard
+disk drive. A part can be assigned to a single device at a time. You can move
+parts from one device to another when you need it.
+
+
+Adding Assets
+-------------
+
+If you want add data center asset you need to fill a few required fields.
+Look at the screen below (you can zoom-in the image in your browser if you
+want) The fields in bold are required. Rest of fields are optional.
+
+.. image:: _static/add-data-center-asset-form.png
+    :scale: 30%
+
+Let's describe each of the fields:
+
+- **Basic info**:
+    - **Type** - a read only field for data center (example: )
```

Comment:
*(example: )* what? similar below  IMHO it's better leave it out alltogether if it's empty for now